x-pack/plugins/ml/public/application/util/ml_error.ts

```diff
@@ -7,16 +7,21 @@
 import { KbnError } from '../../../../../../src/plugins/kibana_utils/public';
 
 export class MLRequestFailure extends KbnError {
-  origError: any;
-  resp: any;
   // takes an Error object and and optional response object
-  // if error is falsy (null) the response object will be used
-  // notify will show the full expandable stack trace of the response if a response object is used and no error is passed in.
+
   constructor(error: any, resp: any) {
```

Comment:
I wonder if we can be more explicit here with the `error: any` since if error obj is required and if `.message` is undefined, the error message shown to the user will be blank.

Reply:
Updated in https://github.com/elastic/kibana/pull/76743/commits/430dbdb5da30a5bb27f20e979e026378afdfcb66
I've also broken the error processing file into multiple files and moved the `MLRequestFailure` into common and changed it so it is no longer a subclass of `KbnError`, as it didn't gain us anything and meant it had to live in public.